deployments/quick-start/kubernetes-vault-rbac.yaml

```diff
@@ -0,0 +1,110 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kube-vault
+  namespace: default
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: kube-vault
+rules:
+- apiGroups: [""]
+  resources:
+  - pods
+  verbs: ["get", "list"]
```

Comment:
Yes, kubernetes-vault only needs these 2 operations:

- Watch for new pods so that it knows pods are created. Specifically: `/api/v1/watch/pods`. It does supports watching namespaces using a regular expression, so it does that filtering internally. I haven't had time to play around with Kubernetes' RBAC too much, but I wonder if it would break if we use RBAC to restrict it to certain namespaces. For example, using RBAC, I restrict it to the `default` namespace, will the watch at `/api/v1/watch/pods` still work correctly expect it will only return events in the `default` namespace?

- List endpoints, specifically: `/api/v1/namespaces/{namespace}/endpoints`.